### PR TITLE
show related entities of bill actions

### PR DESCRIPTION
**Pull request by fgregg**
This PR updates the Bill Detail view to show related entities to a bill action. We need this for municipal scrapers so we can track which committee has current control of a bill see https://github.com/opencivicdata/scrapers-us-municipal/issues/46#issuecomment-107647420

### CODE REVIEW

File: imago/serialize.py [hunk 1]
```diff
@@ -268,7 +271,9 @@ def sfilter(obj, blacklist):
 ])
 
 
+
 BILL_SERIALIZE['related_bills']['bill'] = BILL_SERIALIZE
+#BILL_SERIALIZE['actions']['related_entities']['entity'] = {"name" : {}}
```
Comment (mileswwatkins) [inline imago/serialize.py]: Is this comment supposed to be removed, or is it clarifying something?

Comment (fgregg) [inline imago/serialize.py]: Woops, it should be removed.

**Comment by fgregg** [comment]
@mileswwatkins, I removed that dead code

**Comment by mileswwatkins** [comment]
Thanks @fgregg, I'll check this PR locally this week, and then merge and deploy it.

**Comment by fgregg** [comment]
Thanks!

File: imago/serialize.py [hunk 2]
```diff
@@ -268,6 +271,7 @@ def sfilter(obj, blacklist):
 ])
 
 
+
```
Comment (jpmckinney) [inline imago/serialize.py]: Can you remove this extra line as well? And preferably squash the extra commits into the first one: http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html

**Comment by fgregg** [comment]
@jpmckinney commits are squashed!

**Comment by mileswwatkins** [comment]
Thanks @jpmckinney!

**Comment by fgregg** [comment]
@jpmckinney thanks!

@mileswwatkins when will this change be reflected at api.opencivicdata.org

**Comment by fgregg** [comment]
@mileswwatkins any updates on when this might be reflected on api.opencivicdata.org?

**Comment by mileswwatkins** [comment]
@fgregg, sorry for the delay! I'll address this this week.

**Comment by fgregg** [comment]
@mileswwatkins thanks!

**Comment by mileswwatkins** [comment]
@fgregg: up for me, look good to you? Eg,
http://api.opencivicdata.org/ocd-bill/0000171c-16b0-49af-985b-2de407c851ce/

**Comment by fgregg** [comment]
perfect!

**Pull request merged**